src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/book_store?serverTimezone=UTC";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "test");
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Cannot load JDBC driver", e);
+        }
+    }
+
+    public static Connection getConnection() throws SQLException {
```

Comment:
The method `getConnection()` should be surrounded with a try-with-resources block to ensure that the `Connection` object is closed properly after use to prevent potential resource leaks.